src/main/java/teammates/ui/webapi/action/CreateFeedbackResponseCommentAction.java

```diff
@@ -52,18 +52,7 @@ public ActionResult execute() {
         FeedbackResponseAttributes response = logic.getFeedbackResponse(feedbackResponseId);
         Assumption.assertNotNull(response);
 
-        FeedbackResponseCommentSaveRequest comment = getAndValidateRequestBody(FeedbackResponseCommentSaveRequest.class);
-
-        // String giverEmail = response.giver;
```

Comment:
Can the commented code be removed? @wkurniawan07

Reply:
It's fine to remove. As long as I'm aware, I can trace this back somehow.